seismic/receiver_fn/config_pts2grid_example.json

```diff
@@ -4,15 +4,17 @@
         "sandbox/PV-227/ccp_line_data_sample1.csv":
         {
             "weighting": 1.0,
-            "scale_length_km": 25.0
+            "scale_length_degrees": 0.2,
+            "sample_weighting_enabled": false
         },
         "sandbox/PV-227/ccp_line_data_sample2.csv":
```

Comment:
Andrew's first version of Json schema needs to be revised: I do not think "sandbox/PV-227/ccp_line_data_sample1.csv" or   "sandbox/PV-227/ccp_line_data_sample2.csv" should be used as KEY. Instead, I think we should look to use 
"CSV_FILE":  "sandbox/PV-227/ccp_line_data_sample2.csv",

Similar problem with his orientation correction output script.

No problem with merge.
Just keep in mind for future improvement!

Reply:
No worries. I am planning to expand this JSON config to be fed into a workflow that does the point blending then also outputs the cartopy plots, gmt input files and geotiffs/shapefiles. I will fix up the schema as part of this work. I've sent an email about possible format.